canonical-web-and-design/ubuntu.com
# Engage pages from discourse

## Done

- Extended engage pages to use discourse
- Index page uses now discourse
- Individual engage pages have webinar image pull from brighttalk
- Added thank-you pages for the forms and related engage pages.

## QA

- Go to https://ubuntu-com-8086.demos.haus/engage and click the different engage pages
- CTA should not show for large screens, if 2 CTAs should show on large screens. Otherwise it should only display on small screens.
- CTA buttons should link to respective elements on the page
- Fill in some forms and see that they take you to the thank-you page.
- If you have a Brittalk account, login to check that the video plays correctly.

## Issues

Fix https://github.com/canonical-web-and-design/web-squad/issues/3035
Fix https://github.com/canonical-web-and-design/web-squad/issues/3034
